distribution/sender/routes/db-sync-route.xml

```diff
@@ -4,100 +4,68 @@
-
-                    <log message="Entity not found for request with uuid: ${exchangeProperty.event.requestUuid}" />
-                    
-                    <setBody>
-                        <spel>#{new org.openmrs.eip.component.model.SyncModel()}</spel>
-                    </setBody>
-                    <script>
-                        <spel>
-                            #{body.setMetadata(new org.openmrs.eip.component.model.SyncMetadata())}
-                        </spel>
-                    </script>
-                </when>
+                <simple>${exchangeProperty.isValid}</simple>
+                
+                <setProperty name="messageUuid">
+                    <spel>#{getProperty('event').requestUuid != null ? getProperty('event').requestUuid : T(java.util.UUID).randomUUID().toString()}</spel>
+                </setProperty>
                 
+                <setBody>
+                    <spel>#{new org.openmrs.eip.app.management.entity.SenderSyncMessage()}</spel>
+                </setBody>
                 <script>
                     <spel>
-                        #{body.metadata.setSourceIdentifier('{{db-sync.senderId}}')}
-                        #{body.metadata.setDateSent(T(java.time.LocalDateTime).now())}
-                        #{body.metadata.setOperation(getProperty('event').operation)}
-                        #{body.metadata.setRequestUuid(getProperty('event').requestUuid)}
+                        #{body.setTableName(getProperty('event').tableName)}
+                        #{body.setIdentifier(getProperty('event').identifier)}
+                        #{body.setOperation(getProperty('event').operation)}
+                        #{body.setSnapshot(getProperty('event').snapshot)}
+                        #{body.setDateCreated(new java.util.Date())}
+                        #{body.setMessageUuid(getProperty('messageUuid'))}
                     </spel>
                 </script>
-                <setBody>
-                    <method beanType="org.openmrs.eip.component.utils.JsonUtils" method="marshall(${body})" />
-                </setBody>
-
-                <log loggingLevel="DEBUG" message="Sync payload -> ${body}" />
-
-                <when>
-                    <simple>{{openmrs.eip.dbsync.encryption.enabled}} == true</simple>
-                    <log message="Encrypting entity payload.." />
-                    
-                    <process ref="pgpEncryptService" />
-
-                    <log message="Encrypted entity payload -> ${body}" loggingLevel="TRACE" />
-                </when>
                 
-                <log message="Sending entity to sync destination: {{camel.output.endpoint}}" />
-
-                <toD uri="{{camel.output.endpoint}}" />
-
-                <log message="Entity sent!" />
+                <log message="Saving sender sync message: ${body}" loggingLevel="DEBUG" />
+        
+                <to uri="jpa:SenderSyncMessage?usePersist=true" />
```

Comment:
According to the [documentation](https://camel.apache.org/components/3.15.x/jpa-component.html), we have:

```
usePersist (producer) 
Indicates  to use entityManager.persist(entity) instead of  entityManager.merge(entity). Note: entityManager.persist(entity) doesn’t  work for detached entities (where the EntityManager has to execute an  UPDATE instead of an INSERT query)!.
```

Since we are creating a new entity to the database here **and not updating** by any chance, I think that the appropriate instruction to use is `persist` instead of `merge`.